### DISCUSSION TITLE
user_id refresh problem

### DISCUSSION THREAD

**Discussion by abhinavdawar** [emailjs-com/help#30] [Q&A]
I am using emailjs and it is really awesome.
But am facing a problem now. In every few hours my public_key which has to be kept in. user_id is refreshing. Please solve this as soon as possible.
Thankyou !!

**Comment by xr0master** [comment]
This is possible if you manually refresh the API keys, and it should not happen automatically.
Let's see if anyone else has a similar issue.

**Reply by abhinavdawar** [reply]
It is happening automatically.  After some time keys are getting refreshed
